### PR TITLE
Bug 1150567. Fix fx family nav text overlap.

### PR DESCRIPTION
**Pull request by jpetto**
- Fix overlapping when clicking V2 top level items
- Fix overlapping when hovering then tab focusing

### CODE REVIEW

File: media/css/firefox/family-nav.less [hunk 1]
```diff
@@ -237,7 +238,7 @@
     }
 
     .primarynav {
-        a {
+        .primary-link {
```
Comment (craigcook) [inline media/css/firefox/family-nav.less]: Maybe add a <code>cursor: pointer;</code> for :hover? They're no longer links but the spans are still interactive and it feels weird to get the default insert cursor.

File: media/css/firefox/family-nav.less [hunk 2]
```diff
@@ -785,4 +787,28 @@ html[lang^='en'] .fxfamilynav .subsubnav a {
...
+                }
+            }
+
+            .primary-link:focus + .subnav {
+                display: none;
+            }
+        }
+
+        .subnav.active {
+            display: block !important;
```
Comment (craigcook) [inline media/css/firefox/family-nav.less]: Is this <code>!important</code> justified? I think it's needed to override inline styles applied by JS, correct?

File: media/css/firefox/family-nav.less (hunk 1 above)
Comment (jpetto) [inline media/css/firefox/family-nav.less]: Ah, yeah, good call. Will add.

File: media/css/firefox/family-nav.less (hunk 2 above)
Comment (jpetto) [inline media/css/firefox/family-nav.less]: It unfortunately is, as it needs to override the `display: none;` values directly above, which in turn are overriding the no JS `display` values set prior.

**Comment by craigcook** [comment]
LGTM! r+wc! WTG FTW BBQ! :pig:

File: media/css/firefox/family-nav.less (hunk 2 above)
Comment (craigcook) [inline media/css/firefox/family-nav.less]: That's what I thought, just making sure it wasn't a leftover from testing or something.

**Pull request merged**